src/protect/mutation_calling/fusion.py

```diff
@@ -13,6 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from __future__ import print_function
+import collections
+import csv
+import logging
+from math import ceil
```

Comment:
To maintain symmetricity, import in this order:
```
from __future__ import ...
from system_package import ...

from bd2k_package import ...

import system_package

import bd2k package
```
Each group is alphabetically sorted.